checks.d/zk.py

```diff
@@ -210,4 +317,34 @@ def parse_stat(self, buf):
         _, value = buf.readline().split(':')
         metrics.append(('zookeeper.nodes', long(value.strip())))
 
-        return metrics, tags, mode
+        return metrics, tags, mode, version
+
+
+    def parse_mntr(self, buf):
+        ''' `buf` is a readable file-like object
+            Returns: a tuple (metrics, mode)
+            if mode == 'inactive', metrics will be None
+        '''
+
+        buf.seek(0)
+        first = buf.readline() # first is version string or error
+        if first == 'This ZooKeeper instance is not currently serving requests':
+            return (None, 'inactive')
+
+        metrics = {}
+
+        for line in buf:
+            data = line.split()
+            if len(data) == 2:
+                name = data[0].replace('zk', 'zookeeper').replace('_', '.')
```

Comment:
Tiny nitpick, but would we want to be careful about inadvertently replacing `zk` from the middle of the string? Also can you split out this logic into a `_normalize_metric_label` function so it's easier to test and play around with?